src/Compatibility/Core/src/Android/AppCompat/TabbedPageRenderer.cs

```diff
@@ -1011,7 +1011,10 @@ ColorStateList GetColorStateList(int defaultColor, int checkedColor)
 			states[1] = GetEmptyStateSet();
 			colors[1] = defaultColor;
 
+#pragma warning disable RS0030
+			//TODO: port this usage to Java, if this becomes a performance concern
 			return new ColorStateList(states, colors);
+#pragma warning restore RS0030
```

Comment:
I did not port `TabPage` to use the new methods. Shell should be more common, anyway?

We can fix this in the future if needed.